.github/workflows/triage.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: Add ticket to project
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/add-to-project@v0.3.0
+      - uses: actions/add-to-project@v1
```

Comment:
I think we should actually remove this triage workflow. It was created before github's board have the ability to automatically add issues to the board. I think that it can be replaced with some built-in board's features

Reply:
Per f2f conversation, @francocipollone will take a look at this.

Reply:
We need to continue using add-to-project gh action.
Workflows in boards are allow to add issues/pr to project boards but we need a workflow per repository and for our github plan we only can do one.